nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQL.java

```diff
@@ -463,6 +463,39 @@ public class CaptureChangeMySQL extends AbstractSessionFactoryProcessor {
                     SSL_MODE_VERIFY_IDENTITY)
             .build();
 
+    public static final PropertyDescriptor KEEP_ALIVE = new PropertyDescriptor.Builder()
+            .name("Keep Alive")
+            .displayName("Keep Alive")
+            .description("Used to specify heartbeat messages or queries to be sent after a connection has been idle for a period of time to ensure that " +
+                    "the connection remains active. This prevents the database server or middleware from closing the " +
+                    "connection on a connection that has been inactive for a long time")
+            .required(false)
+            .allowableValues("true", "false")
+            .defaultValue("true")
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor KEEPALIVE_INTERVAL = new PropertyDescriptor.Builder()
+            .name("KeepAlive Interval")
+            .displayName("KeepAlive Interval")
+            .description("Set the Keep-Alive interval in milliseconds for the connection. the default is 60 seconds")
+            .required(false)
+            .addValidator(StandardValidators.POSITIVE_LONG_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
+            .defaultValue("60000")
```

Comment:
Instead of using a number of milliseconds, this property can use the Time Period Validator, which supports different expressions, such as `60 s` or `500 ms`. That provides a more flexible configuration approach, which allowing the implementation code to get the value as milliseconds either way.